resources/js/components/fieldtypes/assets/AssetsFieldtype.vue

```diff
@@ -351,6 +352,10 @@ export default {
             return this.config.show_set_alt && ! this.isReadOnly;
         },
 
+        canBrowse() {
+            return this.can('view '+ this.container +' assets')
```

Comment:
This should also take into account the `configure asset containers` permission:

```suggestion
            return this.can('configure asset containers') || this.can('view '+ this.container +' assets')
```